README-InstrumentQ.md

```diff
@@ -62,9 +86,19 @@ to pick parameter names different from the following (unless you want to use tho
 ### Instruments
 
 Select the instruments that you want to use in the Instrument Query. Each instrument will be treated
-like a separate database table and every record from the first instrument will be joined to every
-row in the second instrument. You can limit which rows are included in the join using the ON
-condition logic. For example: `[instrument_1][record_id] = [instrument_2][record_id]`
+like a separate database table and every row from the first instrument will be joined to every
+row in the second instrument.
+
+You can limit which rows are included in the join using the condition logic. For example:
+`[instrument_1][record_id] = [instrument_2][record_id]`. When you add new instruments, condition
+logic will automatically be added so only rows with matching record IDs will be joined.
+
+Two types of join are supported. **Inner join** will simply join every row in the new instrument
+with every row in the preceding data (subject to condition logic). This means that if you are
+joining two instruments where one has 5 rows and the other has 10 rows, then without condition logic

```

Comment:
not sure if this is clear description of inner join to a non-sql-speaker.  How about "This means that if you are joining two instruments, only those rows with linking fields that match in both instruments will be returned."

Reply:
Amended